src/common/TALENTS/priest.ts

```diff
@@ -448,13 +453,13 @@ const talents = createTalentList({
   VOID_ERUPTION_TALENT: {
     id: 228260,
     name: 'Void Eruption',
-    icon: 'spell_priest_void_blast',
+    icon: 'spell_priest_void-blast',
```

Comment:
This should still be `spell_priest_void_blast`.
```suggestion
    icon: 'spell_priest_void_blast',
```

Reply:
I've undone all of the changes to TALENTS